src/client/components/side-drawer/side-drawer.tsx

```diff
@@ -16,7 +16,9 @@ export interface SideDrawerProps extends React.Props<any> {
   onOpenAbout: Fn;
   onClose: Fn;
   customization?: Customization;
-  showOverviewLink?: boolean;
+  isHome?: boolean;
+  isCube?: boolean;
+  isLink?: boolean;
```

Comment:
That's what I thought, but then I realized I'd have to somehow share the constants (`CUBE`, `LINK`, etc.)... sooooo I'm keeping that for later ^^